operator/docs/lokistack/object_storage.md

```diff
@@ -38,6 +38,30 @@ _Note_: Upon setting up LokiStack for any object storage provider, you should co
       --from-literal=region="<AWS_REGION_YOUR_BUCKET_LIVES_IN>"
     ```
 
+    or with `SSE-KMS` encryption
+
+    ```console
+    kubectl create secret generic lokistack-dev-s3 \
+      --from-literal=bucketnames="<BUCKET_NAME>" \
+      --from-literal=endpoint="<AWS_BUCKET_ENDPOINT>" \
+      --from-literal=access_key_id="<AWS_ACCESS_KEY_ID>" \
+      --from-literal=access_key_secret="<AWS_ACCESS_KEY_SECRET>" \
+      --from-literal=sse_type="SSE-KMS" \
+      --from-literal=sse_kms_key_id="<AWS_SSE_KMS_KEY_ID>" \
```

Comment:
Do we want to mention anywhere that the "KMS Key Id" can be either the ARN of the key or just the UUID? (as a hint for the user)

Reply:
We should reference the AWS docs here instead of replicating validation inputs. WDYT?

Reply:
I was thinking of making it "more easily discoverable", because at least for me the AWS docs can be a mess. But just linking to the docs is also an alternative.